src/server/endpoints/covidcast_utils/model.py

```diff
@@ -299,3 +311,263 @@ def map_row(source: str, signal: str) -> str:
         return signal_source.source
 
     return transformed_pairs, map_row
+
+
+def _resolve_all_signals(source_signals: Union[SourceSignalPair, List[SourceSignalPair]], data_sources_by_id: Dict[str, DataSource]) -> Union[SourceSignalPair, List[SourceSignalPair]]:
+    """Expand a request for all signals to an explicit list of signal names.
+
+    Example: SourceSignalPair("jhu-csse", signal=True) would return SourceSignalPair("jhu-csse", [<list of all JHU signals>]).
+    """
+    if isinstance(source_signals, SourceSignalPair):
+        if source_signals.signal == True:
```

Comment:
I suppose it doesn't matter, because of the way `_parse_common_multi_arg` is coded, signal is a bool if and only if it is equal to True. 

This happens if someone makes a request for `signal=jhu-csse:*` (in theory, though currently the API seems to be spinning endlessly on this simple query `https://api.covidcast.cmu.edu/epidata/covidcast/?signal=jhu-csse:*&time=day:20220102&geo_type=state&geo_values=ca`)